src/isMobile.ts

```diff
@@ -129,6 +132,7 @@ export default function isMobile(userAgent?: string): isMobileResult {
   };
 
   result.any =
+    result.apple.universal ||
```

Comment:
not needed when apple.universal is included in apple.device

Reply:
+1